src/index.html

```diff
@@ -7,8 +7,33 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a class="logo_link"
+         href="#Home"
+         alt="Home">
+        <img  src="./images/logo.png" alt="logo">
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li class="list_item"><a class="nav_link is-active" href="#Apple">Apple</a></li>
+          <li class="list_item"><a class="nav_link" href="#Samsung">Samsung</a></li>
+          <li class="list_item"><a class="nav_link" href="#Smartphones">Smartphones</a></li>
+          <li class="list_item">
+            <a class="nav_link"
+             href="#Laptops_Comp"
+             data-qa="hover">
+             Laptops & Computers
+            </a>
+          </li>
+          <li class="list_item"><a class="nav_link" href="#Gadgets">Gadgets</a></li>
```

Comment:
```suggestion
          <li class="list_item"><a class="nav_link" href="#gadgets">Gadgets</a></li>
```

Reply:
Better will be to start them with small letter
P.S. But remember about formatting, as was written above^